app/controllers/hub/documents_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
     end
 
     def show
-      render_active_storage_attachment @document.upload
+      redirect_to transient_storage_url(@document.upload.blob, disposition: :inline)
```

Comment:
If inline is the fallback default, you could get away with passing it in here!

Reply:
Good call! Removed.